cmd/juju/commands/upgrademodel_test.go

```diff
@@ -136,7 +136,7 @@ var upgradeJujuTests = []upgradeTest{{
 	currentVersion: "2.0.0-ubuntu-amd64",
 	agentVersion:   "2.0.0",
 	args:           []string{"--agent-version", "5.2.0"},
-	expectErr:      `"5.2.0" is not a supported version`,
+	expectErr:      `cannot upgrade to "5.2.0"`,
```

Comment:
We still need the explanation of why we cannot upgrade to 5.2.0, ie it is not supported.
The change to the message removes the reason why and so the user is left without an explanation